### PR TITLE
Binder credit reqs

### PR DESCRIPTION
**Pull request by callummole**
Updated credit requests submitted Dec 2021 and March 2022 to support the Turing cluster of the binder federation.

### CODE REVIEW

**Comment by malvikasharan** [comment]
@KirstieJane can you have a look at this PR?

**Comment by callummole** [comment]
@sgibson91 can you review and merge before I merge #2340?

File: project_management/credit_requests/turing_mybinder_azure_credit_request_2022-03.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Turing Binder Federation Membership - Credit Request
+
+## Project Title
+
+Turing membership of the International Binder Federation
+
+## Project PI
+
+Callum Mole; cmole@turing.ac.uk
```
Comment (sgibson91) [inline project_management/credit_requests/turing_mybinder_azure_credit_request_2022-03.md:9]: It might be useful to also list @malvikasharan here?

**Review by sgibson91 — APPROVED** [review]

File: project_management/credit_requests/turing_mybinder_azure_credit_request_2022-03.md (hunk 1 above)
Comment (callummole) [reply project_management/credit_requests/turing_mybinder_azure_credit_request_2022-03.md:9]: @sgibson91 have done so.

**Pull request merged**